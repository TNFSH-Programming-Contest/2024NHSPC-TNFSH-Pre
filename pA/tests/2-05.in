597254403369912974 644053090077115233 324941058518235350
609437819291379649 389064953407171149 567745779266712759
