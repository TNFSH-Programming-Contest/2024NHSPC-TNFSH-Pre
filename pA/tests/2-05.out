1472616032973127543
